narwhal/consensus/src/tests/bullshark_tests.rs

```diff
@@ -377,98 +373,6 @@ async fn missing_leader() {
-
-    let initial_committee = ReconfigureNotification::NewEpoch(committee.clone());
-    let (tx_reconfigure, rx_reconfigure) = watch::channel(initial_committee);
-
-    let store = make_consensus_store(&test_utils::temp_dir());
-    let cert_store = make_certificate_store(&test_utils::temp_dir());
-    let gc_depth = 50;
-    let metrics = Arc::new(ConsensusMetrics::new(&Registry::new()));
-    let bullshark = Bullshark::new(committee.clone(), store.clone(), gc_depth, metrics.clone());
-
-    let _consensus_handle = Consensus::spawn(
-        committee.clone(),
-        store,
-        cert_store,
-        rx_reconfigure,
-        rx_waiter,
-        tx_primary,
-        tx_consensus_round_updates,
-        tx_output,
-        bullshark,
-        metrics,
-        gc_depth,
-    );
-    tokio::spawn(async move { while rx_primary.recv().await.is_some() {} });
-
-    // Run for a few epochs.
-    for epoch in 0..5 {
-        // Make certificates for rounds 1 and 2.
-        let genesis = Certificate::genesis(&committee)
-            .iter()
-            .map(|x| x.digest())
-            .collect::<BTreeSet<_>>();
-        let (mut certificates, next_parents) =
-            test_utils::make_certificates_with_epoch(&committee, 1..=2, epoch, &genesis, &keys);
-
-        // Make two certificate (f+1) with round 3 to trigger the commits.
-        let (_, certificate) = test_utils::mock_certificate_with_epoch(
-            &committee,
-            keys[0].clone(),
-            3,
-            epoch,
-            next_parents.clone(),
-        );
-        certificates.push_back(certificate);
-        let (_, certificate) = test_utils::mock_certificate_with_epoch(
-            &committee,
-            keys[1].clone(),
-            3,
-            epoch,
-            next_parents,
-        );
-        certificates.push_back(certificate);
-
-        // Feed all certificates to the consensus. Only the last certificate should trigger
-        // commits, so the task should not block.
-        while let Some(certificate) = certificates.pop_front() {
-            tx_waiter.send(certificate).await.unwrap();
-        }
-
-        // Ensure the first 4 ordered certificates are from round 1 (they are the parents of the committed
-        // leader); then the leader's certificate should be committed.
-        let committed_sub_dag = rx_output.recv().await.unwrap();
-        let mut sequence = committed_sub_dag.certificates.into_iter();
-        for _ in 1..=4 {
-            let output = sequence.next().unwrap();
-            assert_eq!(output.epoch(), epoch);
-            assert_eq!(output.round(), 1);
-        }
-        let output = sequence.next().unwrap();
-        assert_eq!(output.epoch(), epoch);
-        assert_eq!(output.round(), 2);
-
-        // Move to the next epoch.
-        committee.epoch = epoch + 1;
-        let message = ReconfigureNotification::NewEpoch(committee.clone());
-        tx_reconfigure.send(message).unwrap();
-    }
-}
-
```

Comment:
I've removed all the tests that test changing epoch / committee in place in NW, but left the ones that test shutdown and restart